### PR TITLE
doc: generator code health (2)

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: utils/doclint/generateApiJson.js [hunk 1]
```diff
@@ -62,6 +62,7 @@ function serializeMember(member) {
   const result = { ...member };
   sanitize(result);
   result.args = {};
+  delete member.clazz;
```
Comment (dgozman) [inline utils/doclint/generateApiJson.js:65]: Put this to sanitize()?

File: utils/generate_types/index.js [hunk 2]
```diff
@@ -239,6 +260,12 @@ function parentClass(classDesc) {
 
 function writeComment(comment, indent = '') {
   const parts = [];
+  
+  comment = comment.replace(/\[`([^`]+)`\]\(#([^\)]+)\)/g, '[`$1`](https://github.com/microsoft/playwright/blob/master/docs/api.md#$2)');
```
Comment (dgozman) [inline utils/generate_types/index.js:264]: Should we have a shared place for all these magic md regexes scattered around the files?

**Review by dgozman — APPROVED** [review]

File: utils/doclint/generateApiJson.js (hunk 1 above)
Comment (pavelfeldman) [reply utils/doclint/generateApiJson.js:65]: Sure.

File: utils/generate_types/index.js (hunk 2 above)
Comment (pavelfeldman) [reply utils/generate_types/index.js:264]: This code is volatile, I don't think sharing these makes sense.

File: utils/doclint/generateApiJson.js (hunk 1 above)
Comment (pavelfeldman) [reply utils/doclint/generateApiJson.js:65]: (follow up)

**Pull request merged**